apps/frontend/src/screens/Game.tsx

```diff
@@ -306,10 +315,12 @@ export const Game = () => {
               </div>
             </div>
             <div className="rounded-md pt-2 bg-bgAuxiliary3 flex-1 overflow-auto h-[95vh]">
-              {!started && (
-                <div className="pt-8 flex justify-center w-full">
+              {!started? (
```

Comment:
What is this, not getting this condition.

why we show 
```
<div className="p-8 flex justify-center w-full">
                  <ExitGameModel onClick={() => handleExit()} />
                </div>
```
                
 if not started

Reply:
If the game is not started the Play button and Waiting loader will render, if it's started the Exit Button will render.
The <div> you have mentioned above is the component of Exit button.

Please let me know if it seems complex at first glance, i'll replace the ternary with if else.